src/common/meta/src/heartbeat/utils.rs

```diff
@@ -56,3 +57,62 @@ pub fn outgoing_message_to_mailbox_message(
         )),
     })
 }
+
+/// Extracts datanode workloads from the provided optional `NodeWorkloads`.
+///
+/// Returns default datanode workloads if the input is `None`.
+pub fn get_datanode_workloads(node_workloads: Option<&NodeWorkloads>) -> DatanodeWorkloads {
```

Comment:
is this function unused? can't find its call site

Reply:
No, just search the function name.